### ISSUE TITLE
AttributeError: 'ApplicationClient' object has no attribute 'ServiceBindings'

### ISSUE THREAD

**Issue by kumbachamushroom** [ibm-watson-iot/iot-python#171]
When I try to create a new cloudant service with cloudantService = appClient.serviceBindings.create(serviceBinding) I get the error AttributeError: 'ApplicationClient' object has no attribute 'ServiceBindings'

**Comment by durera** [comment]
What version of the SDK are you using?  Looks to be fine in the current, and soon to be released 0.10, releases:

```
>>> import wiotp.sdk.application
>>> myConfig = wiotp.sdk.application.parseEnvVars()
>>> client = wiotp.sdk.application.ApplicationClient(config=myConfig, logHandlers=None)
>>> print(client.serviceBindings)
<wiotp.sdk.api.services.ServiceBindings object at 0x007422B0>

```

**Comment by fsmunoz** [comment]
Version 0.11 doesn't work for me... perhaps the API has changed, but if so all the documentation (namely the one around Watson IoT and Cloudant) is using the previous one.
```
fsmunoz:~$ pip3.8 list wiotp-sdk
Package           Version  
----------------- ---------
certifi           2020.12.5
chardet           4.0.0    
idna              2.10     
iso8601           0.1.13   
paho-mqtt         1.5.1    
pip               19.3.1   
pytz              2021.1   
PyYAML            5.4.1    
requests          2.25.1   
requests-toolbelt 0.9.1    
setuptools        41.6.0   
urllib3           1.26.3   
wiotp-sdk         0.11.0   
fsmunoz:~$ cat wt.py 
import wiotp.sdk.application
myConfig = wiotp.sdk.application.parseEnvVars()
client = wiotp.sdk.application.ApplicationClient(config=myConfig, logHandlers=None)
print(client.serviceBindings)
fsmunoz:~$ python3.8 wt.py
Traceback (most recent call last):
  File "wt.py", line 4, in <module>
    print(client.serviceBindings)
AttributeError: 'ApplicationClient' object has no attribute 'serviceBindings'
fsmunoz:~$
